chemprop/nn/predictors.py

```diff
@@ -71,16 +74,21 @@ def __init__(
         dropout: float = 0,
         activation: str = "relu",
         criterion: LossFunction | None = None,
+        output_transform: OutputTransform | None = None,
     ):
         super().__init__()
-        self.save_hyperparameters()
+        self.save_hyperparameters(ignore=["output_transform"])
         self.hparams["cls"] = self.__class__
 
         self.ffn = MLP(
             input_dim, n_tasks * self.n_targets, hidden_dim, n_layers, dropout, activation
         )
         self.criterion = criterion or self._default_criterion
 
+        if output_transform is None:
+            output_transform = OutputTransform([0] * n_tasks, [1] * n_tasks)
+        self.output_transform = output_transform
```

Comment:
two options here (neither is strictly better):
1.
```suggestion
        if output_transform is None:
            output_transform = nn.Identity()
        self.output_transform = output_transform
```
2. get rid of the `if`-statement here and down in `forward()`:
```python
    def forward(self, Z: Tensor) -> Tensor:
        Y_hat = self.ffn(Z)
        if self.output_transform is not None:
            Y_hat = output_transform(Y)

        return Y_hat
```